doc/dev/debugging.rst

```diff
@@ -181,22 +181,22 @@ to perform. Most of the standard passes are described in the
 implemented in Catalyst and can be found in the sources.
 
 All pipelines are executed in sequence, the output MLIR of each non-empty pipeline is stored in
-memory and becomes available via the :func:`~.print_compilation_stage` function in the ``debug`` module.
+memory and becomes available via the :func:`~.get_compilation_stage` function in the ``debug`` module.
```

Comment:
These cross-references need updating actually since the function is no longer accessible top-level, but in the `debug` module.